cuebot/src/main/java/com/imageworks/spcue/service/HostManagerService.java

```diff
@@ -164,6 +164,12 @@ public DispatchHost createHost(RenderHost rhost, AllocationEntity alloc) {
         hostDao.tagHost(host, alloc.tag, HostTagType.ALLOC);
         hostDao.tagHost(host, host.name, HostTagType.HOSTNAME);
 
+        if (rhost.getTagsCount() > 0) {
+            for (String tag : rhost.getTagsList()) {
+                hostDao.tagHost(host, tag, HostTagType.MANUAL);
```

Comment:
Should we also update the gRPC schema to allow RQD to specify Manual & Hardware tags?  Or are we OK defaulting to manual?

Reply:
Based on the comment below, I would default to manual.